lib/elastic-search/index-schema.js

```diff
@@ -183,6 +187,7 @@ exports.schema = () => ({
       },
       idBarcode: propertyTemplates.exactString,
       location: propertyTemplates.entity,
+      m2CustomerCode: propertyTemplates.exactString,
```

Comment:
were these fields just not being indexed till now?

Reply:
They were being indexed; They were not mentioned in the `index-schema` file, which wasn't really being used yet.